corehq/apps/app_manager/views/apps.py

```diff
@@ -126,6 +127,9 @@ def default_new_app(request, domain):
         app.secure_submissions = True
     clear_app_cache(request, domain)
     app.save()
+    if toggles.GUIDED_TOUR.enabled(domain):
+        tours.SIMPLE_NEW_APP.setup(request, domain)
```

Comment:
The notable difference between this and toggles is that the toggles API is immutable (except for the `set` method, which is only used in tests—a smell in itself), while this new tours API is not.